.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="GoogleStyle" />
```

Comment:
Just out of interest, how does this work. I dont have the google style formatting installed by default?

Reply:
i assume the pre-req here would be to install https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

Reply:
Yep, I think that's the point, people probably weren't aware of importing the GoogleStyle scheme into IntelliJ (https://google.github.io/styleguide/intellij-java-google-style.xml)
They weren't in version control - so I think the project xml codeStyles can be removed if Tony has exported the GoogleStyle scheme into the `.editorconfig` and further customised as seen in our current file e.g. to correct the java imports